packages/playwright-test/src/runner.ts

```diff
@@ -251,14 +273,13 @@ export class Runner {
         throw new Error(`Cannot find project group '${projectGroup}' in the config`);
       for (const entry of group) {
```

Comment:
This whole method could move to `RunPhase`?

Reply:
Done.